apps/mobile/src/components/SafeListItem/SafeListItem.tsx

```diff
@@ -109,6 +116,24 @@ export function SafeListItem({
   )
 }
 
+const ProposalBadge = function ProposalBadge() {
+  return (
+    <Badge
+      circular={false}
+      content={
+        <View alignItems="center" flexDirection="row" gap="$1">
+          <SafeFontIcon size={12} name="info" />
+
+          <Text fontWeight={600} color={'$color'}>
+            Proposal
+          </Text>
+        </View>
+      }
+      themeName="badge_background"
+    />
+  )
+}
+
```

Comment:
could we put this into a separate file? It's not per see related to SafeListItem.